presto-main/src/main/java/io/prestosql/sql/gen/LambdaBytecodeGenerator.java

```diff
@@ -142,6 +142,14 @@ public static CompiledLambda preGenerateLambdaExpression(
                 lambdaExpression);
     }
 
+    /**
+     * Parameter name for the generated method.
+     */
+    private static String parameterName(int argumentPosition, String lambdaArgumentName)
+    {
+        return "lambda_" + argumentPosition + "_" + lambdaArgumentName.replaceAll("[^a-zA-Z_]", "");
```

Comment:
I realize `"lambda_" + argumentPosition` is enough from generated code perspective.
I retained argument name in case there is any need to debug the generated code.

Reply:
@Praveen2112 was there a similar escaping somewhere already? I think you did one, but I can't remember where was it.

Reply:
Yes !! It was in scalar function .. #2081

Reply:
Thanks, updated